GitUp/Application/Document.m

```diff
@@ -338,7 +338,9 @@ - (void)windowControllerDidLoadNib:(NSWindowController*)windowController {
   _searchResultsViewController.emptyLabel = NSLocalizedString(@"No Results", nil);
   [_searchControllerView replaceWithView:_searchResultsViewController.view];
 
-  _quickViewController = [[GIQuickViewController alloc] initWithRepository:_repository];
+  _quickViewModel = [[QuickViewModel new] configuredWithRepository:_repository];
```

Comment:
Make this an alloc init. I don't want `new` in this codebase lol. Make the initializer take the repository and don't have a separate `config` method.